channels/br.m3u

```diff
@@ -315,3 +315,33 @@ http://unitvaovivo.ufrgs.br:8080/live.ogg
 https://srv5.zcast.com.br/andre4369/andre4369/playlist.m3u8
 #EXTINF:-1 tvg-id="Yeeaah.br" tvg-name="Yeeaah!" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/TxdOyKS.png" group-title="General",Yeeaah! (720p)
 https://srv3.zcast.com.br/yeeaah/yeeaah/playlist.m3u8
+#EXTINF:-1 tvg-id="Loading.br" tvg-name="Loading" tvg-country="BR" tvg-language="Portuguese" tvg-logo="" group-title="",Loading (1080p) [Geo-blocked]
+https://google.gcp.br.cdn.booyah.live/hls/151073/master_34850987.m3u8
```

Comment:
Geo-blocked ? Works for me.
![vlcsnap-2021-05-31-17h41m37s334](https://user-images.githubusercontent.com/30985701/120216552-76612580-c237-11eb-9a96-630333eeb1ba.png)

Reply:
Yeah, you're right. Apparently only the player was blocked.